pyproject.toml

```diff
@@ -1,13 +1,13 @@
 [project]
 name = "drizzle"
 description = "A package for combining dithered images into a single image"
-requires-python = ">=3.8"
+requires-python = ">=3.9"
```

Comment:
is this necessary? if so, remove tests for 3.8

Reply:
I would argue that we should not impose requirement to increase the Python version if the code can run in the old version of Python (with an older `numpy` or whatever). Maybe we do not need to run unit tests agains Python 3.8 (so no official support for older versions)?